vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -487,7 +487,7 @@ - (void)reportLoadedPageWithURL:(NSURL *)url innerText:(NSString *)text tabId:(N
 {
   if (![self isAdsServiceRunning]) { return; }
   const auto urlString = base::SysNSStringToUTF8(url.absoluteString);
-  ads->OnPageLoaded((int32_t)tabId, urlString, urlString, base::SysNSStringToUTF8(text));
+  ads->OnPageLoaded((int32_t)tabId, { urlString }, base::SysNSStringToUTF8(text));
```

Comment:
Does iOS need to implement this correctly?

Reply:
It does and we were not sure if this was possible so it was left for investigation. I have raised https://github.com/brave/brave-ios/issues/3111

Reply:
Assuming we don't support it, will there be any specific issues?

Reply:
If iOS is unable to support then ad conversions will not work for some advertisers who require the intermediate URLs